bin/get-docs.sh

```diff
@@ -14,12 +14,10 @@ cd $deps/renovate
 
 if [ -z $SKIP_BUILD ]; then
```

Comment:
I copy/pasted the current bash shell script from this PR branch into the ShellCheck website [^ShellCheck] which finds the following problems:

```sh
Line 8:
rm -rf $docs
       ^-- SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
rm -rf "$docs"
 
Line 9:
mkdir -p $docs
         ^-- SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
mkdir -p "$docs"
 
Line 13:
cd $deps/renovate
   ^-- SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
cd "$deps"/renovate
 
Line 15:
if [ -z $SKIP_BUILD ]; then
        ^-- SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
if [ -z "$SKIP_BUILD" ]; then
 
Line 20:
cp -R tmp/docs/* $docs
                 ^-- SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
cp -R tmp/docs/* "$docs"
 
Line 22:
cd $pwd
   ^-- SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
cd "$pwd"
 
Line 25:
echo '---\ntitle: Merge Confidence\ndescription: Node versions support in Renovate\n---\n' > docs/merge-confidence.md
     ^-- SC2028 (info): echo may not expand escape sequences. Use printf.
```

[^shellcheck]: https://www.shellcheck.net/

Reply:
Let's do this in separate PR, please open an issue for this.

Reply:
Okay! Will make a new issue for this. 😉